ideas/172-topic-democracy.md

```diff
@@ -0,0 +1,83 @@
+# Democracy
+
+## Preamble
+
+    Idea: 172-topic-democracy
+    Title: Subjective Delegation Liquid Democracy
+    Status: Draft
+    Created: 2018-04-15
+
+## Summary
+A democracy where you can specify delegate for carbon-voting specific multi level topics or fallback to a predefined defaults if none set.
+
+## Swarm Participants
+- Lead Contributor: Ricardo Guilherme Schmidt
+- Testing & Evaluation: <!-- @username -->
+- Contributor: <!-- @username -->
+- Contributor: <!-- @username -->
+- PM: <!--- @username -->
+- UX (if relevant): <!-- @username -->
+
+## Product Overview
+
+This product intend replace authority centralized control for networks with  tokenized participation, such as Status Network currently owned by MultiSig contract 0xBBF0cC1C63F509d48a4674e270D26d80cCAF6022.
```

Comment:
YES! I remember looking into this and thinking "hm...".

Reply:
Something to change here?